.travis.yml

```diff
@@ -55,6 +66,26 @@ before_install:
   - rm -rf tmp-build
   - if [[ -n $BUILD_TYPE ]]; then bash mk/bootstrap-librdkafka.sh ${LIBRDKAFKA_VERSION} tmp-build ; fi
   - go get -u golang.org/x/lint/golint && touch .do_lint
+  - mkdir -p "$GOPATH/src/gopkg.in/confluentinc/confluent-kafka-go.v1/"
+  - cp -r . "$GOPATH/src/gopkg.in/confluentinc/confluent-kafka-go.v1/"
+  - |-
+      case $TRAVIS_OS_NAME in
```

Comment:
Put the stuff below in mk/Makefile and call it from here, or a source script in mk/